spec/index.html

```diff
@@ -10424,7 +10424,205 @@ <h3>Evaluation Semantics</h3>
+              </table>
+              <div class="note">
+                <p>
+                  This restriction means that <a href="#defn_valuesinsertion">values inserted</a>
+                  do not conflict with values assigned to variables within the pattern.
+                </p>
+                <p>
+                  This operation is performed as part of query parsing.
+                </p>
+              </div>
+            </section>
+            <section>
+              <h4>Remapping</h4>
+              <p>
+                Remapping ensures that a variable name used inside a project expression,
+                but which is not part of the results of the evaluation of the project expression,
+                does not coincide with a variable mentioned anywhere else in the
+                algebra expression of a query.
+              </p>
+              <p>
+                Renaming these variables does not change the results of evaluating
+                the project expresssion.
+              </p>
+              <div class="defn">
+                <b>Definition: <span id="defn_projmap" name="defn_projmap">Projection Expression Variable Remapping</span></b>
+                <p>
+                  For a projection algebra operation #sparqlProjection `Project(A, PV)` acting on algreg express `A` and with set of variables `PV`, define
+                  a partial mapping `F` from
+                  `<a href="#sparqlQueryVariables">V</a>`,
+                  the set of all variables, to `V` where:
+                </p>
+                <pre>F(v) = v1 if v is in PV, where v1 is a fresh variable
+F(v) = v if v is not in PV</pre>
+                <p>
+                Define the Projection Expression Variable Remapping `ProjectMap(P, PV)`
+                </p>
+                <pre>ProjectMap(Project(A, PV)) = Project(A1, PV) 
+                                where A1 is the result of applying F
+                                to every variable mentioned in A.
+                </pre>
+                <p>
+                  The Projection Expression Variable Remapping yields an algrebra expression that
+                  evaluates to the same results as the Project argument. No variable of `ProjectMap(Project(A, PV))`
+                  that is not in `PV` is mentioned anywhere else in the algebra expression for the query.
+                </p>
+              </div>
+              <p>This process is applied throughout the graph pattern of <code>EXISTS</code>:</p>
+              <div class="defn">
+                <b>Definition: <span id="defn_varrename" name="defn_varrename">Variable Remapping</span></b>
+                <p>
+                  For any algebra expression `X`, define the Variable Remapping `PrjMap(X)` 
+                  of algebra expression `X`:
+                </p>
+                  <pre>PrjMap(X) = replace all project operations Project(P, PV) 
+                with ProjectMap(P, PV) for each projection in X.</pre>
```

Comment:
Is the `ProjectMap(P, PV)` here actually meant to be `ProjectMap( Project(P, PV) )` as introduced in the previous definition? If not, how is `ProjectMap(P, PV)` defined?